### DISCUSSION TITLE
When a 'Dropdown Menu' is active the rest of the page is inaccessible

### DISCUSSION THREAD

**Discussion by alanggdev** [tailwindlabs/headlessui#3385] [Help]
Hi, i'm using headlessui/react 2.1.2
I have a simple Dropdown Menu (just like the example) but I have a problem, if this menu is active the rest of the page is not clickeable until the Menu closes, for example if the menu is opened and I want to click to other component outside the menu, the first click closes the menu andIi have to make another click to interact with the other component.

I think this happend because my all elements in html have _aria-hidden="true" inset_ when the Dropdown Menu is active but when it closes this tags disappear.

I have already tried to use _modal={false}_ but the menus still open if i click other menu or other headless ui component like dialog.

![Screenshot_2](https://github.com/user-attachments/assets/b7e22a6b-ec56-42e3-8a30-cd87c9a8f226)

This behavior did not occur in v1.7.19 but I really dont want to drowngrade, i like the new version

**Comment by corpulent** [comment]
I solved it by adding "modal={false}" to my <MenuItems>. Tailwind is at 3.4.17, headlessui/react at 2.2.9.
